query/appservice/appservice_use_virtual_service_endpoint.sql

```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
+  env.id as resource,
+  case
+    when internal_load_balancing_mode like '%Microsoft.Web%' then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when internal_load_balancing_mode like '%Microsoft.Web%'  then env.name || ' assoicated with "Microsoft.Web" service endpoint.'
```

Comment:
We need to check `Vnet Connection` for app service ,for this we need to add the api call `Get Vnet Connection` in  table `azure_app_web_app`. Please check this once.
```
"policyRule": {
      "if": {
        "allOf": [
          {
            "field": "type",
            "equals": "Microsoft.Web/sites"
          },
          {
            "field": "kind",
            "like": "app*"
          }
        ]
      },
      "then": {
        "effect": "[parameters('effect')]",
        "details": {
          "type": "Microsoft.Web/sites/virtualNetworkConnections",
          "existenceCondition": {
            "field": "Microsoft.Web/sites/virtualnetworkconnections/vnetResourceId",
            "exists": "true"
          }
        }
        ```